plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/utils/RegexUtils.java

```diff
@@ -0,0 +1,18 @@
+package ru.sbtqa.tag.api.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RegexUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=84&issues=AWVaY-0viZQY_aXE-Npd&open=AWVaY-0viZQY_aXE-Npd)

[ISSUE_KEY]: AWVaY-0viZQY_aXE-Npd